### PR TITLE
Added css to keep canvas sized correctly in test. #8406

### PR DESCRIPTION
**Pull request by angrycat9000**
Added the CesiumWidget.css to the testing page.
Added a test to validate that the canvas stays the same size.

Let me know if this is fix works for [issue 8406](https://github.com/AnalyticalGraphicsInc/cesium/issues/8406) @ggetz @IanLilleyT

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @angrycat9000! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/angrycat9000/cesium/blob/issue-8406b/CONTRIBUTORS.md) file!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/issue-8406b/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Specs/SpecRunner.html [hunk 1]
```diff
@@ -21,14 +21,17 @@
         if (window.location.search.indexOf('built') !== -1) {
             if (window.location.search.indexOf('release') !== -1) {
                 document.write('<script src="../Build/Cesium/Cesium.js"><\/script>');
+                document.write('<link rel="stylesheet" href="../Build/Cesium/Widgets/CesiumWidget/CesiumWidget.css">');
```
Comment (mramato) [inline Specs/SpecRunner.html]: There's a top-level widgets.css that imports all widget styles, probably better to use that throughout.

**Comment by angrycat9000** [comment]
Hi @IanLilleyT,

Are you able to provide any insight into these two points:
* The new test fails the headless test in CI, but passes for me in Chrome and Firefox when the tests are run through the browser.
* It is generates a developer error message about the object being destroyed.  Is there something wrong with this test setup?  (and is this possibly causing the first problem)?

Thanks!

**Comment by lilleyse** [comment]
> The new test fails the headless test in CI, but passes for me in Chrome and Firefox when the tests are run through the browser.

I'm not sure... @mramato do you know?

**Comment by lilleyse** [comment]
@angrycat9000 In case this PR gets merged before https://github.com/AnalyticalGraphicsInc/cesium/pull/8407, can you add yourself to `CONTRIBUTORS.md`?

**Comment by angrycat9000** [comment]
Just updated contributors. Thanks!

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @angrycat9000! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by hpinkos** [comment]
@lilleyse @angrycat9000 what's the status of this PR?  Is this ready to be merged?

**Comment by lilleyse** [comment]
Latest was it passed in the browser but not in headless CI. @mramato do you have any ideas?

**Comment by hpinkos** [comment]
closing due to inactivity. Feel free to reopen this PR when we have a chance to get back to it!

**Pull request closed without merging**